qiskit/quantum_info/states/statevector.py

```diff
@@ -104,11 +104,17 @@ def __eq__(self, other):
             self._data, other._data, rtol=self.rtol, atol=self.atol)
 
     def __repr__(self):
-        text = self.draw('text', prefix='Statevector(')
-        return str(text) + ')'
+        prefix = 'Statevector('
+        pad = len(prefix) * ' '
+        return '{}{},\n{}dims={})'.format(
+            prefix, np.array2string(
+                self._data, separator=', ', prefix=prefix),
+            pad, self._op_shape.dims_l())
 
     def draw(self, output=None, max_size=16, dims=None, prefix='', **drawer_args):
-        """Returns a visualization of the Statevector.
+        """Return a visualization of the Statevector.
```

Comment:
Same comments as with the density matrix class for kwargs, import ipython display